WcaOnRails/app/views/competitions/_competition_form.html.erb

```diff
@@ -216,6 +216,13 @@
         <%= f.input :event_restrictions %>
         <%= f.input :event_restrictions_reason %>
 
+        <% if @competition.events.any? %>
+          <hr>
+          <%= f.input :main_event do %>
+            <%= f.select :main_event, options_for_select(@competition.events.map { |event| [event.name, event.id] }, @competition.main_event), {}, { class: "form-control main-event" } %>
```

Comment:
Does this generate a `<select> `with a blank option? I don't think we should default to anything here, and should instead force the user to pick an event.

Reply:
It doesn't add a blank option as is. By forcing the user to pick an event, do you mean adding a validation? I thought about adding one but only after the information for past competitions has been filled, as if not we'll have problems. I could add a validation now similar to https://github.com/thewca/worldcubeassociation.org/blob/master/WcaOnRails/app/models/competition.rb#L759-L761 for instance if you prefer.

Reply:
Oh, I meant that we should not default to selecting a main event id for the user, but should rather default to no event selected. I think that by default, `options_for_select` does not include a blank option. (this is similar to the problem over in https://github.com/thewca/worldcubeassociation.org/pull/4503)

Reply:
Ah, gotcha. I'm adding a blank option.